### PR TITLE
Include policy

### PR DESCRIPTION
**Pull request by tKabiawu**
Added a policy field in the Entry struct.

### CODE REVIEW

**Review by nharper — COMMENTED** [review]
Can you provide some context of where/how this will be used?

File: chromium/preloadlist/preloadlist.go [hunk 1]
```diff
@@ -54,12 +54,16 @@ type PreloadList struct {
 //
 // - Mode: The only valid non-empty value is ForceHTTPS
 //
-// - IncludeSubDomains: If Mode == ForceHTTPS, forces HSTS to apply to
-//   all subdomains.
+// - IncludeSubDomains: If Mode == ForceHTTPS, forces HSTS to apply to all subdomains.
```
Comment (nharper) [inline chromium/preloadlist/preloadlist.go:57]: nit: don't change the formatting of these lines

File: chromium/preloadlist/preloadlist.go [hunk 2]
```diff
@@ -99,7 +103,7 @@ func (idx IndexedEntries) Get(domain string) (Entry, HstsPreloadEntryFound) {
 			return entry, AncestorEntryFound
 		}
 	}
-	return Entry{"", "", false}, EntryNotFound
+	return Entry{"", "", false, "custom"}, EntryNotFound
```
Comment (nharper) [inline chromium/preloadlist/preloadlist.go:106]: This function returns an all-zero Entry if nothing is found. The policy should be "" to keep it returning an all-zero Entry.

**Review by nharper — COMMENTED** [review]
preloadlist_test.go also has some literal Entry structs that should have policies added to them. (Check that `go vet github.com/chromium/hstspreload/...` doesn't return any issues.)

File: chromium/preloadlist/preloadlist.go (hunk 1 above)
Comment (tKabiawu) [reply chromium/preloadlist/preloadlist.go:57]: Sorry about that, I'm not actually changing any of these lines though haha. Whenever I save the document, it auto formats these lines for some reason but I'll try and fix that

File: chromium/preloadlist/preloadlist.go [hunk 3]
```diff
@@ -201,4 +206,4 @@ func NewFromFile(fileName string) (PreloadList, error) {
 	}
 
 	return Parse(b)
-}
+}
```
Comment (tKabiawu) [inline chromium/preloadlist/preloadlist.go:209]: Not sure what this change is reflecting

File: chromium/preloadlist/preloadlist_test.go
Comment (tKabiawu) [inline chromium/preloadlist/preloadlist_test.go:1]: Added `""` to the Entries for the new Policy field

File: chromium/preloadlist/preloadlist.go (hunk 3 above)
Comment (nharper) [reply chromium/preloadlist/preloadlist.go:209]: This is indicating that there's no newline at the end of the file.

**Review by nharper — APPROVED** [review]
looks good, just need to fix the (lack of) newline at the end of the file.

In a future PR, it'd be nice to have tests that check the Policy field gets filled correctly.

**Pull request merged**